examples/carbon-relay-ng.ini

```diff
@@ -58,6 +58,72 @@ init = [
      #'addRoute sendFirstMatch analytics regex=(Err/s|wait_time|logger)  graphite.prod:2003 prefix=prod. spool=true pickle=true  graphite.staging:2003 prefix=staging. spool=true pickle=true'
 ]
 
+# Blacklist
+
+blacklist = [
+  'prefix collectd.localhost',
+  'regex ^foo\..*\.cpu+'
+]
+
+# Aggregations
+
+[[aggregation]]
+type = 'sum'
+regex = '^stats\.timers\.(app|proxy|static)[0-9]+\.requests\.(.*)'
+format = 'stats.timers._sum_$1.requests.$2'
+interval = 10
+wait = 20
+
+[[aggregation]]
+type = 'avg'
+regex = '^stats\.timers\.(app|proxy|static)[0-9]+\.requests\.(.*)'
+format = 'stats.timers._avg_$1.requests.$2'
+interval = 5
+wait = 10
+
+# Rewriters
+
+[[rewriter]]
+old = 'testold'
+new = 'testnew'
+max = -1
+
+# Routes
+
+[[route]]
+key = 'carbon-default'
+type = 'sendAllMatch'
+# prefix = ''
+# substr = ''
+# regex = ''
+destinations = [
+  '127.0.0.1:2003 spool=true pickle=false'
```

Comment:
I didn't see any good/clean way to handle nested structured fields in the config format, and at least we've gotten rid of the double-space issue.  If you have any ideas for a structured config format I'd definitely be open to it, but tend to agree that it's fine this way.